website/content/docs/connect/gateways/api-gateway/configuration/http-route.mdx

```diff
@@ -0,0 +1,605 @@
+---
+layout: docs
+page_title: HTTP Route Configuration
+description: Learn how to configure an HTTP Route bound to an API Gateway on VMs.
+---
+
+# HTTP Route Configuration Reference
+
+This topic provides reference information for the gateway routes configuration entry. Refer to [Consul API Gateway Configuration Overview]() for information about configuring API gateways in Kubernetes environments.
+
+## Configuration model
+
+The following list outlines field hierarchy, language-specific data types, and
+requirements in an `http-route` configuration entry. Click on a property name
+to view additional details, including default values.
+
+- [`Kind`](#kind): string | must be `http-route`
+- [`Name`](#name): string | no default
+- [`Meta`](#meta): map | no default
+- [`Rules`](#rules): list | no default
+- [`Filters`](#rules-filters): map | no default
+  - [`Headers`](#rules-filters-headers): list | no default
+    - [`Add`](#rules-filters-headers): map | no default
+    - [`Remove`](#rules-filters-headers): list | no default
+    - [`Set`](#rules-filters-headers): map | no default
+  - [`URLRewrites`](#rules-filters-urlrewrites): list | no default
+    - [`Path`](#rules-urlrewrites-path): map | no default
```

Comment:
@trujillo-adam is correct that this is missing a number of fields, also with the latest change for `URLRewrite` (loss of plural and flattened to a single entry v. a list)
```suggestion
  - [`URLRewrite`](#rules-filters-urlrewrite): map | no default
    - [`Path`](#rules-urlrewrite-path): string | no default
```